jcache/remote/src/main/java/org/infinispan/jcache/remote/RemoteCacheWrapper.java

```diff
@@ -15,7 +15,6 @@
 import org.infinispan.client.hotrod.RemoteCacheManager;
```

Comment:
can you check if this class `RemoteCacheWrapper` can be replaced by `DelegatingRemoteCache`?

Reply:
I had thought of doing it, but never looked into it. I can try real quick.

Reply:
Unfortunately the `DelegatingRemoteCache` requires the new `InternalRemoteCache`. I will just leave it as is for now. We can try to clean up JCache sometime later maybe :D